vignettes/wflow-06-gitlab.Rmd

```diff
@@ -0,0 +1,147 @@
+---
+title: "Hosting worklowr using GitLab"
+subtitle: "workflowr version `r utils::packageVersion('workflowr')`"
+author: "John Blischak"
```

Comment:
This is a substantial contribution. We should discuss adding you as a contributor.

Reply:
Happy to do that if you think it's worth it! 😸

Reply:
Great! I was thinking of 1) listing you as a co-author of this vignette and 2) adding you as a contributor to the package:

```
person("Luke", "Zappia", role = c("ctb"),
       comment = "Instructions for hosting with GitLab")
```

Would that work for you?

Reply:
Yeah, I'm more than happy with that. Thanks!